workflow/roles-and-permissions.rst

```diff
@@ -0,0 +1,113 @@
+
+Reviewer
+~~~~~~~~
+
+A Reviewer role picks up where the Editor leaves off. While a Reviewer does not have as many rights as the Editor, the Reviewer can publish content that has been sent to the submit for publication state or send it back to the owner. The Reviewer also has a special portlet just for content that needs to be reviewed. Once an Editor has submitted content for publication, the Reviewer will review the content and then has the option to Publish or send back the content for the Contributor to review. The Reviewer has the final say if something gets published or not.
+
+Manager
+~~~~~~~
+
+The Manager role is the role that can do everything. A user with the Manager role is a Site Administrator. Manager privileges are not given out lightly as this role can add, delete, and make changes to any thing in the site. While more than one person should have this role, it definitely should not be handed out to large numbers of people. Your site Manger has access to the control panel, where many site wide settings can be changed and updated. The Manager can also manage things via the ZMI (Zope Management Interface).
+
+Giving out permissions
+++++++++++++++++++++++
+
+The easiest way to hand out permissions is to assign roles to groups. You can create a group and assign that group a role. Then, whenever you want to give someone certain permissions, you can add that user to that group. Assigning roles on a group level allows you to more easily manage large numbers of users.
+
+
+Permissions
++++++++++++
+
+Plone’s security system is based on the concept of *permissions* protecting *operations* (like accessing a view, viewing a field, modifying a field, or adding a type of content) that are granted to *roles*, which in turn are granted to *users* and/or *groups*. In the context of developing content types, permissions are typically used in three different ways:
+
+* A content type or group of related content types often has a custom
+  *add permission* which controls who can add this type of content.
+* Views (including forms) are sometimes protected by custom
+  permissions.
+* Individual fields are sometimes protected by permissions,
+  so that some users can view and edit fields that others can’t see.
+
+It is easy to create new permissions. However, be aware that it is considered good practice to use the standard permissions wherever possible and use *workflow* to control which roles are granted these permissions on a per-instance basis.
+
+Standard permissions
+--------------------
+
+The standard permissions can be found in ``Product.Five``\’s ``permissions.zcml`` (``parts/omelette/Products/Five/permissions.zcml``). Here, you will find a short ``id`` (also known as the *Zope 3 permission id*) and a longer ``title`` (also known as the *Zope 2 permission title*). For historical reasons, some areas in Plone use the id, whilst others use the title.
```

Comment:
Product.Five has on ly one permission: `five.ManageSitefive.ManageSite` which is probably never ever used. I'd point to Product.CMFCore instead and mention that many packages define permissions.